.github/CODEOWNERS

```diff
@@ -8,10 +8,13 @@
 # -------------------------------------------------
 
 # -------------------------------------------------
-# All files except a few here are owned by dev team
+# All files are owned by dev team
 # -------------------------------------------------
 
 * @freecodecamp/dev-team
+
+# --- Owned by none (negate rule above) ---
+
 *.md
 package.json
 package-lock.json
```

Comment:
```suggestion
```
I don't think the negation is happening.  If you got to https://github.com/freeCodeCamp/freeCodeCamp/blob/main/LICENSE.md it's still owned by dev-team.

Reply:
Thanks for taking the time to review. 

Firstly, this only works when the change (this PR) lands on main. 

Secondly negating things CODEOWNERS file is bit of a blackhole as seen in this old discussion: https://github.community/t/1423, we won't know until this PR lands.

Reply:
Right, but the existing CODEOWNERS file already uses the filename without an owner pattern. If it worked then `package.json`, `LICENSE.md` and so on wouldn't have owners, but they're all owned by dev-team.

Reply:
This PR adds the change so that they will be accounted for. Like I said it's a blackhole, if you have suggestions as an update to this PR, I am all ears.

Reply:
Yes and no.  We could just try merging this and seeing if it works.

I'm not 100% clear on the objective, though.  Presumably we want to be able to auto-merge renovate PRs, but if someone creates a PR that modifies the package or package-lock, we'd want a dev-team review before it can go in.

For the md files, it seems fine to have dev-team as owners (or staff, maybe?).

Reply:
We have several checks in place including multiple levels of human reviews for each commit that goes into the `prod-current` branch (which is what matters 🤷🏽)

Finally, the current merge requirements (Unfortunately, I do not have the time to explain those) are enough for catching anything out of the ordinary.

IMHO, we have [quite a few things to worry about](https://github.com/freeCodeCamp/freeCodeCamp/issues?q=is%3Aissue+is%3Aopen+sort%3Acreated-asc+-label%3A%22status%3A+on+the+roadmap%22) at the minute.

The objective of this PR is to draw attention to things where it is needed. The above list of issues is one such place to start.

Reply:
RIghto. I'm happy to try this approach and see if it does what we want.